src/systems/bit-media-frames.js

```diff
@@ -102,26 +114,38 @@ function isColliding(world, eidA, eidB) {
   return false;
 }
 
+// TODO we only allow capturing media-loader so rely on its bounds calculations for now
+function getEntityBounds(world, target) {
```

Comment:
Rename `target` to `targetEid` as, IIRC, you suggested to add `Eid` suffix to name of variable that  holds eid?